website/docs/r/apikey.html.markdown

```diff
@@ -47,7 +47,7 @@ See [the NS1 API docs](https://ns1.com/api#getget-all-account-users) for an over
 The following arguments are supported:
 
 * `name` - (Required) The free form name of the apikey.
-* `key` - (Required) The apikeys authentication token.
+* `key` - (Returned upon API Key Creation) The apikeys authentication token.
```

Comment:
If this is not an argument that can be passed to the resource, this should move to the `Attributes` section, below.  That should prob look like:
```
## Attributes Reference

In addition to all arguments above, the following attributes are exported:

* `key` - (Computed) The apikeys authentication token.
```

Reply:
@mburtless agreed.  Updating the PR.  Hadn't seen any other examples of this so wasn't sure it was the right convention.